PACKAGE.md

```diff
@@ -0,0 +1,106 @@
+# Building the package
+## TL;DR
+
+The package is only available at `obs://systemsmanagement:SCC/suseconnect-ng`
+and it can be fetched with `osc` in the usual way. There is no way to update the
+version locally or to try out the a locally generated package with changes on
+the code in an automated manner.
+
+## Step 1. Update version
+
+The version of `connect-ng` is a mix of a version specified in the `.spec` file
+and the git commit sha. The version is thus updated automatically on every git
+commit.
+
+## Step 2. The OBS package
+
+The RPM is built from OBS, and you need to access the main repository from
+there:
+[systemsmanagement:SCC/suseconnect-ng](https://build.opensuse.org/package/show/systemsmanagement:SCC/suseconnect-ng).
+From this repository, you can build the package as usual with:
+
+```bash
+$ osc build openSUSE_Leap_15.4 x86_64 --no-verify
+$ osc build SLE_15_SP3 x86_64 --no-verify
+```
+
+If you want to build the package locally, you will have to compress the code in
+a `.tar.xz` format and match the version that the `.spec` file is expecting.
+This way you will be able to install and test the RPM locally.
+
+## Step 3. Update package version
+
+The package is updated manually and it relies on the Git repository from
+`connect-ng` to contain the latest changes. Whenever you want to update the
+package on OBS, simply run the service and commit the changes like so:
+
+```bash
+$ osc service manualrun
+# review the changes
+$ osc commit
+```
+
+## Submit maintenance updates
```

Comment:
Could we link to generic docs about this as nothing here is specific to this package?

Reply:
This is to mimic what we already have on SUSEConnect ruby, and we have found it quite useful both for newcomers new also to OBS, and for experts since it's a way to double check that no steps are omitted and be more confident.